web/src/lib/components/admin-page/settings/machine-learning-settings/machine-learning-settings.svelte

```diff
@@ -101,10 +102,7 @@
             min={0.001}
             max={0.1}
             desc="Maximum distance between two images to consider them duplicates, ranging from 0.001-0.1. Higher values will detect more duplicates, but may result in false positives."
-            disabled={disabled ||
-              !config.machineLearning.enabled ||
-              !config.machineLearning.clip.enabled ||
-              !config.machineLearning.duplicateDetection.enabled}
+            disabled={disabled || $featureFlags.duplicateDetection}
```

Comment:
Shouldn't this be `!$featureFlags.duplicateDetection`?

Reply:
Most certainly